package.json

```diff
@@ -31,28 +31,29 @@
     "main": "can-stache-bindings"
   },
   "dependencies": {
+    "can-attribute-encoder": "^0.3.0",
```

Comment:
we should hard-code core deps that are not `>1.0` (we don't do this everywhere (can-stache-key), but we should)